README.md

```diff
@@ -33,7 +33,7 @@ After running this code, `foo` will be a `Foo` where `foo.array == [1, 3]` and `
 In your Package.swift:
 ```swift
   dependencies: [
-    .package(name: "ResilientDecoding", url: "https://github.com/airbnb/ResilientDecoding.git", from: "0.9.0"),
```

Comment:
Seems like at minimum we need to update https://github.com/airbnb/ResilientDecoding/blob/58272e7a467c5ea6b78219faf0e54a3987d00fdd/ResilientDecoding.podspec#L3

Reply:
We do need to update the this line.